src/ui.dropdown-badge/UDropdownBadge.vue

```diff
@@ -121,7 +128,7 @@ function onClickOption(option) {
 
     <UDropdownList
       v-if="isShownOptions"
-      ref="dropdownListRef"
+      ref="dropdown-list"
```

Comment:
why in `kebab-case`?

Reply:
It's written this way in vue docs, so i guess it's right way to name it
https://vuejs.org/guide/essentials/template-refs#accessing-the-refs